### ISSUE TITLE
Make sure existing Home Screen detox tests are working, before HSP XL is merged into develop

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#8102]
This ticket is not adding new functionality (ex: tests for the new home screen buttons)

This ticket is making sure that all pre-existing detox tests still work on the HSP branch. I will list out which ones we changed (ex: removing the COVID-19 portions in ticket #7853) & do any work not caught before the initial ticket moved into the XL, on this ticket.

Covered by this ticket:
* Home screen script
* Login, signin and messages scripts -- small change for the veterans crisis line component
* Updates to Availability Framework and Nav scripts to account for the above

NOT covered by this ticket:
* Veterans status card (existing script, ticket XXXX)
* Net-new scripts for HSP (ticket #8103)

**Comment by TKDickson** [comment]
[Running Android e2e on the current XL branch, just to kind of get a starting baseline.](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/runs/8117119449)

**Comment by TKDickson** [comment]
Home screen, navigation, category screen, VSC (home and profile)

**Comment by TKDickson** [comment]
Started, at least, [a branch](https://github.com/department-of-veterans-affairs/va-mobile-app/tree/feature/8102-TKD-existing-detox-fixes-HSP) for this work.

**Comment by TKDickson** [comment]
Abandoned that very old branch and opened a new one - here's [the codespace psychic disco](https://github.com/codespaces/psychic-disco-94vgwqxgggqcqx5). I realized very quickly that I'm blocked by the "make cerner optional" ticket (have added as a blocker for this one), so am going to pivot to starting to get new homescreen functionality detox tests written, instead.

**Comment by TKDickson** [comment]
Login and signin are working (changes for veterans crisis line component) - here's [a run](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/runs/9647172902) where they passed for Android (the only platform that's working with detox right now).

Still working on home (down to two non-passing cases which I'm trying to get local setup in order to debug more efficiently), and it looks like messages and VSC are not going to pass in that same run linked above, TBD what's broken and if it's related to HSP.

**Comment by TKDickson** [comment]
Messages is in the clear in that ^ run too.

**Comment by TKDickson** [comment]
The only case that is still failing (locally) is AF, and that's related to #8907 (it's a bug in the overall detox framework for testing AF, related to "if there's a real AF set up then detox breaks", not a problem with home screen specifically. AKA it's also broken in develop).

I will attach a link to a working detox run for all of the scripts I touched, here & in the PR. Will go kick one off now.

**Comment by TKDickson** [comment]
[TestRail with all the things passing](https://dsvavsp.testrail.io/index.php?/runs/view/5651&group_by=cases:section_id&group_order=asc) (except the one thing covered by the pre-existing bug, #8907)

**Comment by TKDickson** [comment]
Approved and merged, closing
